### DISCUSSION TITLE
Actividad 4 - GRUPO 3

### DISCUSSION THREAD

**Discussion by BogadoLucasDemian** [frlp-utn-ingsoft/catedra#162] [Show and tell]
https://github.com/BogadoLucasDemian/vetsoft/tree/main

**Comment by ivanduva** [comment]
Buenas! Les puedo pedir el link a donde lo deployaron?

**Reply by BogadoLucasDemian** [reply]
Buenas Ivan, 
El link es este: https://vetsoft-1-0.onrender.com/
Me dice que tiene error de servidor, pero probe crear la imagen del docker local y corre todo perfecto, no se que podría ser el error.

https://github.com/frlp-utn-ingsoft/catedra/assets/60678759/aa53c4c1-1576-4794-ae7d-6bde9eb511e6

**Reply by ivanduva** [reply]
Creo que es porque no están haciendo el migrate en el dockerfile.
Al no funcionar la release, está incompleto. Así que arreglen en un PR y después les seguimos corrigiendo.

**La última fecha límite de será el último día de cursada antes de las vacaciones: Lunes 8 de Julio 23:59 hs.**

Saludos

**Reply by BogadoLucasDemian** [reply]
Buenas tardes Ivan,

Efectivamente era eso, muchas gracias. Ya quedó corregido

**Reply by ivanduva** [reply]
Buenas,
Van a tener que arreglar algunas cosas:

- En cuanto a la validación de que el teléfono sea numérico, les pedimos que bloqueen los caracteres no numéricos en el form y que lo testeen con un e2e.
- En cuanto a la validación de que el teléfono empiece con 54, en ese PR dejaron un bug que borra el campo del form al tirar el error. El form debería mantener el valor.
- En cuanto a la validación del email, se permite agregar "@vetsoft.com" sin nada adelante. Arreglen eso.
- Luego de hacer todo, saquen una release. Piensen bien que número de versión correspondería y además fijense que cuando documentan la relase títulos como "Vetsoft con CD arreglado" o "Quinta versión de vetsoft", no son explicativos. En general los títulos de las releases son los números de las versiones sumando algun dato como la fecha y tienen un subtítulo más explicativo como por ejemplo: "Fix de migraciones en la CD" o "Agrega city en Cliente". Como ejemplo, fijense como lo hacen en el repositorio de React: https://github.com/facebook/react

Recuerden que los cambios deben ser realizados por las personas que se encargaron de cada issue.

Por otro lado, seguimos viendo problemas en las reviews. Más allá de la poca interacción, hay algunas malas prácticas como por ejemplo, en este PR: https://github.com/BogadoLucasDemian/vetsoft/pull/29 solucionan problemas del PR anterior. Eso se evita justamente haciendo una buena review y solucionando esos problemas en el mismo PR a través de comentarios. También vimos un revert que no terminamos de entender la razón. Lo mismo, con una buena review se evitan estos inconvenientes. Igualmente un revert se hace únicamente en un caso muy grave y como última instancia. No creemos que este sea el caso, se podría haber solucionado de otra manera.
Tengan esto en cuenta en las correcciones.

**La última fecha límite de será el último día de cursada antes de las vacaciones: Lunes 8 de Julio 23:59 hs.**

Saludos

**Reply by RoTantos** [reply]
Buenas tardes! Ya hicimos las correcciones. Saludos!

**Comment by ivanduva** [comment]
Buenas!
Perfecto, el trabajo integrador queda aprobado.
Valoramos que hayan agregado algunos tests que no habíamos pedido, que además están muy bien. Sólo un comentario que les dejamos para que les sirva en el futuro, no es necesario que lo arreglen: en este PR https://github.com/BogadoLucasDemian/vetsoft/pull/37 se agrega un test e2e que no hace ningún expect, por lo tanto no testea nada.

Felicitaciones!
Saludos
